client/src/components/chat/Chat.js

```diff
@@ -24,6 +25,12 @@ export class Chat extends React.Component {
 
     configureSocket = () => {
         var socket = socketClient(SERVER);
+
+        socket.emit('test','isOnline');

```

Comment:
Please keep the names more meaningful, you can't make out what you are doing here by emitting a test event.

Reply:
Alternatively, you can even add comments to show what you are doing here.